src/js/common/helpers/login-helpers.js

```diff
@@ -27,6 +27,7 @@ export function getUserData(dispatch) {
       if (userData.first_name) {
         sessionStorage.setItem('userFirstName', userData.first_name);
       }
+      dispatch(updateProfileField('savedForms', json.data.attributes.in_progress_forms));
```

Comment:
This list of dispatches makes me sad. It could be one dispatch call.

(Not that you need to fix that here).